predictive_models/baseline.py

```diff
@@ -0,0 +1,51 @@
+"""A Baseline model
+Source: https://www.tensorflow.org/tutorials/structured_data/time_series
+"""
+
+import tensorflow as tf
+import pandas as pd
+from window_generator import WindowGenerator
+
+
+# pylint: disable=too-few-public-methods
+class Baseline(tf.keras.Model):
+    def __init__(self, label_index=None):
+        super().__init__()
+        self.label_index = label_index
+
+    def call(self, inputs):
+        if self.label_index is None:
+            return inputs
+        result = inputs[:, :, self.label_index]
+        return result[:, :, tf.newaxis]
+
+
```

Comment:
Anything after this should move to a `main` like function, which can be called from the `../main.py`.